### PR TITLE
Prefetch Updates: `open_sub_manual_acking()`

### PR DESCRIPTION
**Pull request by ric-evans**
Add special logic for rabbitmq to get around its prefetch limits (https://www.rabbitmq.com/consumer-prefetch.html). Removes the `use_prefetch_value` arg from `open_sub_manual_acking()`: see https://github.com/Observation-Management-Service/MQClient/blob/v2.3.1/mqclient/queue.py#L322-L327

### CODE REVIEW

**Comment by ric-evans** [comment]
TODO: need tests for retries--will closing the channel affect future acks from the old channel?

**Comment by ric-evans** [comment]
TODO: shuffle acks

File: mqclient/broker_client_interface.py [hunk 1]
```diff
@@ -106,7 +106,8 @@ class RawQueue:
     """Raw queue object, to hold queue state."""
 
     def __init__(self) -> None:
-        pass
+        # can be changed by Queue per situation, then broker client handles accordingly
+        self.connection_can_have_multiple_unacked_messages = False
```
Comment (dsschult) [inline mqclient/broker_client_interface.py:110]: Can you comment on what this means, since it looks like this is currently always False?

**Review by dsschult — COMMENTED** [review]
Generally looks good to me.

**Review by dsschult — APPROVED** [review]

**Pull request merged**